### PR TITLE
Add missing `become: yes` for package installation

### PR DESCRIPTION
**Pull request by cfelder**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@
   package:
     name: "{{ __podman_packages }}"
     state: present
+  become: yes
```
[the remark this answers is not in the document]
Comment (richm) [reply tasks/main.yml:10]: The podman role is designed in such a way that you specify `podman_run_as_user` in order to configure podman for a particular user - I suppose another way to run it is to just run it as that user and have the role figure out that it needs to update the user containers/pods/systemd units - does it work that way other than the need to use `become: true` for `package`?  Another way to do it:
```
     become: "{{ true if ansible user is non-root else false }}"
```
but then I'm not sure how to determine if the ansible user is root or not

[the remark this answers is not in the document]
Comment (cfelder) [reply tasks/main.yml:10]: I would like to use this role for creating a rootless container. Imho it should not be required to run this role as root except for installing the podman package or applying firewall rules. Afair this was possible w/ the previous podman-container-systemd role by Ilkka Tengvall.

[the remark this answers is not in the document]
Comment (cfelder) [reply tasks/main.yml:10]: ``become: yes`` should not hurt when being run as root already

[the remark this answers is not in the document]
Comment (richm) [reply tasks/main.yml:10]: > `become: yes` should not hurt when being run as root already

Right.  The other way around this is to ensure that `podman` is installed *before* running the role, either in the playbook or during the provisioning step.  One use case is that I do not want to use sudo *at all*, I just want the role to ensure that the `podman` package is already present - fail if not present, do not install it.  In that case, I would not want to use `become: true`, unless the `package` module is smart enough to check for the presence of the package without using sudo at all.

[the remark this answers is not in the document]
Comment (cfelder) [reply tasks/main.yml:10]: Ok, I understand see #24 for an alternative approach

**Pull request closed without merging**